inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd

```diff
@@ -8,10 +8,11 @@ date: '2022-10-28'
 
 
 # Lab Methods
-CEM.NKR~CCR5~ cells [@trkola1999cell] were 
-used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]). 
+CEM.NKR~CCR5~ cells [@trkola1999cell] were used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]). 
 
 Peripheral blood mononuclear cells (PBMCs) were obtained from a HIV-seronegative donor by leukapheresis and cryopreserved until the day of the assay. After thawing and overnight resting in RPMI 1640 supplemented with antibiotics, $10\%$ fetal bovine serum (R10), and 10 ng/mL of IL-15, the PBMCs were used as effector cells at an effector-to-target ratio of 30:1. 
 
 Target and effector cells were plated in white 96-well half-area plates and co-cultured with 4-fold [Adjust if different] serial dilutions of trial participant serum starting at the 1:50 dilution. For each sample, percent specific killing was measured in duplicate at dilutions of 1:50, 1:200, 1:800, 1:3200, 1:12800, and 1:51200 [Adjust if dilutions are different]. Co-cultures were incubated for 6 hours at $37 ^{\circ} C$ in $5\% \text{ CO}_2$. The final readout was the reduction of luminescence intensity generated by the presence of residual intact target cells that had not been lysed by the effector population in the presence of ADCC-mediating serum antibodies. The percentage of killing was calculated using the formula: percent specific loss Luciferase activity = 100 * (RLU of target and effector well– RLU of test well)/(RLU of target and effector well).
```

Comment:
Target and effector cells were plated in white 96-well half-area plates and co-cultured with trial participant serum sample starting at the 1:50 dilution and 4-fold serial dilutions thereon (1:200, 1:800, 1:3200, 1:12800, and 1:51200) [Adjust if dilutions are different]. Co-cultures were incubated for 6 hours at $37 ^{\circ} C$ in $5\% \text{ CO}_2$. For each serial dilution of the sample, the final readout was measured in duplicate. The final readout was the reduction of luminescence intensity generated by the presence of residual intact target cells that had not been lysed by the effector population in the presence of ADCC-mediating serum antibodies. The percentage of killing was calculated using the formula: percent specific loss Luciferase activity = 100 * (RLU of target and effector well– RLU of test well)/(RLU of target and effector well).

Reply:
Additional comment: 
It is not clear from the text how the duplicates were performed. My understanding is that each sample is split into two, and each of the two splits are serially diluted and final readouts taken. If that is correct, it should help if we mention that explicitly.